infrastructure/libtor/src/tor.rs

```diff
@@ -97,18 +93,21 @@ impl Tor {
         // data dir
         let temp = tempdir()?;
```

Comment:
Why not keep tor data in one place (e.g some path relative to the node's base dir `{base_dir}/tor_data`) instead of a new temp dir each time?

Then tor can reuse this each time and should bootstrap faster.

Reply:
I am going for this option